Solution1/WeatherApi/Extensions/StartupFilterRegistration.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using WeatherApi.Infrastructure;
+
```

Comment:
what if you'll have multiple filters? will you rename all classes and methods?

Reply:
+